app/src/main/java/com/example/ithaca_transit_android_v2/MainActivity.kt

```diff
@@ -59,13 +56,14 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
         setContentView(R.layout.activity_main)
 
         mSearchAdapter = SearchViewAdapter(this, mSearchLocations)
-        mRouteViewAdapter =
-            RouteViewAdapter(this, ArrayList())
+        mRouteListViewAdapter =
+            RouteListViewAdapter(this, ArrayList())
+        mRouteDetailAdapter = RouteDetailAdapter(this, route_detail_data)
         mSearchPresenter = SearchPresenter(search_card_holder, map_fragment as MapFragment,this, mSearchAdapter)
-        mRouteOptionsPresenter = RouteOptionsPresenter(bottomSheet, mRouteViewAdapter)
+        mRouteOptionsPresenter = RouteOptionsPresenter(bottom_sheet, mRouteListViewAdapter, mRouteDetailAdapter, this)
         mRouteOptionsPresenter.setBottomSheetCallback(
-            BottomSheetBehavior.from(bottomSheet),
-            bottomSheet
+            BottomSheetBehavior.from(bottom_sheet),
+            bottom_sheet
```

Comment:
just checking, is this correct?

Reply:
Good catch. this was changed but u right we only need to pass in bottom_sheet now